micropipenv.py

```diff
@@ -973,6 +975,19 @@ def _get_index_entry_str(sections, package_info=None):  # type: (Dict[str, Any],
     return result
 
 
+def resolve_nested_variables(url):
```

Comment:
The function can be private:

```suggestion
def _resolve_nested_variables(url):
```